docs/core-concepts/app-deployment.md

```diff
@@ -1,17 +1,82 @@
-# 🔧 .NET Core Application Deployment concepts
-
-> **Note**
-> 
-> This topic hasn’t been written yet! 
->
-> We welcome your input to help shape the scope and approach. You can
-> track the status and provide input on this
-> [issue](https://github.com/dotnet/core-docs/issues/471) at GitHub.
-> 
-> If you would like to review early drafts and outlines of this
-> topic, please leave a note with your contact information in the issue.
->
-> Learn more about how you can contribute on
-> [GitHub](https://github.com/dotnet/core-docs/blob/master/CONTRIBUTING.md).
->
-	
\ No newline at end of file
+# .NET Core Application Deployment concepts
+.NET Core applications can be deployed in one of two ways:
+
+1. As a portable shared framework application.
+2. As a platform specific standalone application.
+
+##  Shared framework applications
+Shared framework applications take a dependency on the .NET Core shared framework being pre-installed on a machine.  As such they are deemed *portable* since the same application binaries can run on any machine that has the correct version of the shared framework installed.  Shared framework applications are `.dll` files that can be launched by calling `dotnet app.dll`.  Shared framework applications are the default type of application produced by `dotnet new`.
+
+### Sample project
+```
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.0"
+    }
+  },
+  "frameworks": {
+    "netcoreapp1.0": { }
+  }
+}
+```
+The part of this project that distinguishes it from a standalone application is the inclusion of `"type": "platform"` for the `Microsoft.NETCore.App` meta-package.  This tells NuGet that the `Microsoft.NETCore.App` is a platform package that will be provided by the system.  Another distinguishing factor is the lack of the `"runtimes"` section in the project.json.  This will cause NuGet to evaluate all platform specific bits in other packages and include them inside the application in platform specific subfolders.  At runtime the .NET Core host will choose the correct platform specific bits for the platform the app is running on.
+
+## Standalone applications
+Standalone applications take no dependency on .NET Core being preinstalled on a machine.  Instead they carry .NET Core with them as part of the application.  Since .NET Core contains platform specific native binaries standalone applications are platform specific themselves, though they may be published for multiple platforms.  Standalone applications consist of `app.exe`, a renamed version of the platform specific .NET Core host, and `app.dll` the actual application.
```

Comment:
Not clear whether "app.exe" is a literal or a stand-in for the actual app name, i.e., for a project in the helloworld folder would this be helloworld.exe?  The first part of the sentence sounds like it's a literal, the last part ("the actual application) sounds like it's a placeholder.